### ISSUE TITLE
make db_bench problem

### ISSUE THREAD

**Issue by sarsanaee** [facebook/rocksdb#4729]
Hi,

I'm building db_bench with ```make db_bench``` but I got some errors like: 

```
/home/seyedali/testing/rocksdb/tools/db_bench_tool.cc:5739: undefined reference to `google::SetUsageMessage(std::string const&)'
/home/seyedali/testing/rocksdb/tools/db_bench_tool.cc:5742: undefined reference to `google::ParseCommandLineFlags(int*, char***, bool)'
tools/db_bench_tool.o: In function `__static_initialization_and_destruction_0(int, int) [clone .constprop.929]':
/home/seyedali/testing/rocksdb/tools/db_bench_tool.cc:86: undefined reference to `google::FlagRegisterer::FlagRegisterer<std::string>(char const*, char const*, char const*, std::string*, std::string*)'
/home/seyedali/testing/rocksdb/tools/db_bench_tool.cc:196: undefined reference to `google::FlagRegisterer::FlagRegisterer<long>(char const*, char const*, char const*, 
```

I was wondering if I made a mistake in installing gflags but I guess I installed them fine because ```make all``` for examples works OK! And I'm using centos!!

**Comment by ajkr** [comment]
What is the gflags version and what operating system are you using? Also can you provide the output of `ldd ./db_bench`? Thanks.

**Comment by sarsanaee** [comment]
Thank you for the reply!

Here is the output of my uname -a "Linux rachmaninoff2 3.10.0-862.14.4.el7.x86_64 #1 SMP Wed Sep 26 15:12:11 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux"

I just clone the master branch of the gflags and installed it!

**Comment by ajkr** [comment]
Want to take a look at my comment in #4694 and see if that helps? CentOS linking problems come up surprisingly often :/.

**Comment by zcdqs** [comment]
I have the same problem and it is solved by using gcc 4.9(from scl) other than 4.8 of centos 7

**Comment by sarsanaee** [comment]
So it seems that was a compiler version issue! Alright, I close it then!
